Tests/SE0000_KeyPathReflectionTests/SE0000_KeyPathReflectionTests.swift

```diff
@@ -1,12 +1,18 @@
 import XCTest
 @testable import SE0000_KeyPathReflection
 
-private protocol Protocol {}
-extension Int: Protocol {}
+private protocol Protocol {
+  var foo: Int { get }
+}
+extension Int: Protocol {
+  var foo: Int { 0 }
+}
```

Comment:
Any particular reason this requirement was added to the protocol? Maybe accidental?

Reply:
Ah, I was trying to add some protocol existential key path negative tests. I can revert it.